models/preles/R/runPRELES.jobsh.R

```diff
@@ -42,40 +40,85 @@ runPRELES.jobsh<- function(met.file,outdir,start.date,end.date){
   sec = udunits2::ud.convert(sec,unlist(strsplit(nc$dim$time$units," "))[1],"seconds")
   
   ##build day and  year
-  ifelse(leap_year(year)==TRUE,
+  ifelse(leap_year(as.numeric(year))==TRUE,
          dt <- (366*24*60*60)/length(sec), #leap year
          dt <- (365*24*60*60)/length(sec)) #non-leap year
   tstep = 86400/dt
   
   doy <- rep(1:365,each=86400/dt)
-  if(year %% 4 == 0){  ## is leap
+  if(as.numeric(year) %% 4 == 0){  ## is leap
     doy <- rep(1:366,each=86400/dt)
   }
   
   ## Get variables from netcdf file
+  ## Get variables from netcdf file
   PAR <-ncvar_get(nc,"surface_downwelling_photosynthetic_photon_flux_in_air") #PAR in mol/m2s1
   Tair <-ncvar_get(nc,"air_temperature")#air temperature in K
-  Precip <-ncvar_get(nc,"PREC")#precipitation in kg/m2s1
-  VPD <-ncvar_get(nc,"water_vapor_saturation_deficit") # Vapor pressure deficit in Pa
+  Precip <-ncvar_get(nc,"precipitation_flux")#precipitation in kg/m2s1
   CO2 <-ncvar_get(nc,"mole_fraction_of_carbon_dioxide_in_air") #mol/mol
-  
+  RH <- ncvar_get(nc,"relative_humidity")
```

Comment:
FYI, RH is also not the 'standard' variable that's guaranteed to be there, that would be SH. FYI VPD is just the difference between saturation vapor pressure and the actual vapor pressure. You get the former from Tair and pressure (there's a function in metutils), and the latter comes directly from SH.